internal/tiltfile/os/os.go

```diff
@@ -27,6 +28,11 @@ func (e Extension) OnStart(env *starkit.Environment) error {
 		return err
 	}
 
+	err = env.AddBuiltin("os.realpath", realpath)
```

Comment:
should this be os.path.realpath to mirror the python api?

Reply:
When I tried that I got an error saying "multi level modules are not currently supported".

Is that something we control or is that upstream?

Reply:
good question! lemme poke at it